blocked-edges/4.15.8-AWSCustomDomainNodesNotReady.yaml

```diff
@@ -1,5 +1,6 @@
 to: 4.15.8
 from: 4[.]14[.](14|15)
+fixedIn: 4.15.8
```

Comment:
Do you mean `fixedIn: 4.15.9`, since 4.15.9 is the first release with only `4.14.(>=16)` update sources?

Reply:
Yeah, this should be 4.15.9 
/hold

Reply:
Fixed it in https://github.com/openshift/cincinnati-graph-data/commit/035b1090f75e489e65d7f1fabba248f6f7f66baa

Reply:
/hold cancel